src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -201,7 +201,9 @@ class CompetitionRobotContainer {
                 Units.degreesToRadians(5)) // was 2 changed in b80 for wk4
             .withTimeout(0.5));
     NamedCommands.registerCommand("StopShooter", m_Shooter.setSpeed(0));
-    NamedCommands.registerCommand("DriveToNote", new DriveToNote(m_chassis).raceWith(pickUpNote()));
+    NamedCommands.registerCommand(
+        "DriveToNote",
+        new AlignToNote(m_chassis, () -> new ChassisSpeeds(2, 0, 0)).deadlineWith(pickUpNote()));
```

Comment:
I think the pickupNote needs to be the 'deadline', since `AlignToNote` will never end. You could also use `raceWith` since we only have the 2 commands